### ISSUE TITLE
Provide a --details flag for fly teams

### ISSUE THREAD

**Issue by keithkroeger** [concourse/concourse#2215]
# Feature Request

## What challenge are you facing?

We have a need to view details about teams.  For example, the _type_ of authentication and _non-credential_ settings.  Without this detail, in order to audit team access, especially with the new OIDC functionality, we will need to query the teams table to determine some settings.

This would require that only a limited selection of fields are shown from the data stored.

## A Modest Proposal

Add a --details flag to fly teams.

For example, fly might show for some teams
```
> fly -t s teams --details
*name*   *authentication* *details*
main       basic                   username: admin
other      oidc                     users: [acct_one, acct_two], groups: [ad_group_1, ad_group_2]
uaateam uaa
gitlab      gitlab                  group: abcd
...
```

**Comment by jama-pivotal** [comment]
Linking this to #1167 which I entered a while back but got closed. I really thing we might be able to do this once #1888 is done which could address some of the security concerns raised in the previous issue

**Comment by xtremerui** [comment]
This is shipped with concourse 4.0.0. 
`fly -t your-target teams --details` will have an example output

```
name                             users                     groups
some-team                        local:admin               github:some-org
...
```

Closing.
